apps/svelte.dev/content/blog/2025-08-01-whats-new-in-svelte-august-2025.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "What's new in Svelte: August 2025"
+description: 'Async Svelte is out - with all the fixins'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+With [Async Svelte](https://github.com/sveltejs/svelte/discussions/15845) officially out in the world and its companion [Remote Functions](https://github.com/sveltejs/kit/discussions/13897) already ligned up, there's a lot of updates to share!
```

Comment:
Rich just merged and released remote functions 😄 https://github.com/sveltejs/kit/releases/tag/%40sveltejs%2Fkit%402.27.0
```suggestion
With [Async Svelte](https://github.com/sveltejs/svelte/discussions/15845) and its companion [Remote Functions](https://github.com/sveltejs/kit/discussions/13897) officially out in the world, there's a lot of updates to share!
```